pkg/interfaces/contracts/pool-hooks/IMevCaptureHook.sol

```diff
@@ -19,36 +19,36 @@ interface IMevCaptureHook {
     error MevSwapFeePercentageAboveMax(uint256 feePercentage, uint256 maxFeePercentage);
 
     /**
-     * @notice The sender is already registered as MEV tax-exempt.
-     * @param sender Sender that is already MEV tax-exempt
+     * @notice The sender is already registered as MEV capture-exempt.
```

Comment:
Should we rename this? Renaming MevTaxHook to MevCaptureHook makes it more "user-friendly," but it does not imply that the MevCaptureHook should not charge a MEV tax only because the hook's name has changed.

Reply:
Yes; I see the point with "capture-exempt." Maybe the original is enough. I don't feel strongly (depends on the business goal of who asked for it). Feel free to just close this if the original goes far enough. I approved that one.

Reply:
Could also say "priority fee" instead of MEV tax, if that's softer.